challenges/objects-arrays.js

```diff
@@ -89,37 +119,51 @@ The zoo wants to display both the scientific name and the animal name in front o
 
 */
 const animalNames = [];
+
+let fullNames = zooAnimals.forEach(function(i){
+  animalNames.push(i.animal_name + " " + i.scientific_name);
+});
+
 console.log(animalNames);
 
-/* Request 2: .map()    
+/* Request 2: .map()
 
 The zoos need a list of all their animal's names (names only, not scientific) converted to lower case.  Create a new array named lowerCase and map over each name to convert them all to lower case.  Log the resut.
 
 */
 
-const lowerCase = [];
-console.log(lowerCase); 
+let lowerCase = zooAnimals.map((animal) => {
+  return animal.animal_name.toLowerCase();
+});
 
-/* Request 3: .filter() 
+console.log(lowerCase);
+
+/* Request 3: .filter()
 
 The zoos are concenred about animals with a lower population count. Find out which animals have a population less than 5.
 
 */
-const largerPopulation = [];
+
+let largerPopulation = zooAnimals.filter((species) => {
+  return species.population === "5";
```

Comment:
when you have only one parameter in the arrow function you can omit the ( ) around it like so... zooAnimals.filter( species => { return species.population === "5" }